wiki/Beatmap_Spotlights/en.md

```diff
@@ -116,15 +116,17 @@ Rewards are distributed to weekly timeshift lobby winners, creators of curated b
 
 | Badges | Bracket Tier | Placement |
 | :-: | :-- | :-- |
-| ![](img/badges/autumn_2020/osu/ri_2.png) ![](img/badges/autumn_2020/osu/ri_3.png) | Rhythm Incarnate | Top 50 |
-| ![](img/badges/autumn_2020/osu/diamond_2.png) ![](img/badges/autumn_2020/osu/diamond_3.png) | Diamond | Top 51 – 3% |
+| ![](img/badges/autumn_2020/osu/ri_2.png) ![](img/badges/autumn_2020/osu/ri_3.png) | Rhythm Incarnate | The best of the best |
+| ![](img/badges/autumn_2020/osu/diamond_2.png) ![](img/badges/autumn_2020/osu/diamond_3.png) | Diamond | Top 3% |
 | ![](img/badges/autumn_2020/osu/platinum_2.png) ![](img/badges/autumn_2020/osu/platinum_3.png) | Platinum | 3% – 10% |
 | ![](img/badges/autumn_2020/osu/gold_2.png) ![](img/badges/autumn_2020/osu/gold_3.png) | Gold | 10% – 25% |
 | ![](img/badges/autumn_2020/osu/silver_2.png) ![](img/badges/autumn_2020/osu/silver_3.png) | Silver | 25% – 50% |
 | ![](img/badges/autumn_2020/osu/bronze_2.png) ![](img/badges/autumn_2020/osu/bronze_3.png) | Bronze | 50% – 70% |
 | ![](img/badges/autumn_2020/osu/copper_2.png) ![](img/badges/autumn_2020/osu/copper_3.png) | Copper | 70% – 95% |
 | ![](img/badges/autumn_2020/osu/iron_2.png) ![](img/badges/autumn_2020/osu/iron_3.png) | Iron | 95% – 100% |
 
+Rhythm Incarnate tier threshold is picked manually based on season's participant count, it cannot exceed top 50.
```

Comment:
"based on season's participant count" goes for the rest of the tiers too. is there no more accurate way to describe this?

Reply:
can't think of anything else frankly, feel free to suggest something if you have an idea

Reply:
Top 50 or Top #%, whichever is less?

Reply:
calculating a very small % is very painful when it comes to a really small amount of people, the 1% thing wouldn't necessarily be always true as it would need to be adjusted here or there every time the ranking is calculated

Reply:
seems fine to me then, maybe in the future this could be made more specific with more data about what range this usually falls into